test/com/google/javascript/jscomp/integration/ProductionCoverageInstrumentationPassIntegrationTest.java

```diff
@@ -110,18 +78,11 @@ public void testNoTranspilationFunctionInstrumentation() {
 
     String source = lines("function foo() { ", "   console.log('Hello'); ", "}");
 
-    String[] sourceArr = {instrumentCodeSource, source};
-
     String expected =
         lines(
-            "function foo() { ",
-            getInstrumentCodeLine("C", 1, 0),
-            ";",
-            "   console.log('Hello'); ",
-            "}");
+            "function foo() { ", getInstrumentCodeLine("C"), ";", "   console.log('Hello'); ", "}");
```

Comment:
Could you add a trailing `//` comment to make the formatter place the strings on separate lines?

lines(
    "function foo() {", //
     getInstrumentCodeLine("C"),
     ";",
[etc]